api/handler_test.go

```diff
@@ -88,15 +89,21 @@ func getHandler(t *testing.T, blockstoreType string, opts ...testutil.GetDBOptio
 	if blockstoreType == "" {
 		blockstoreType, _ = os.LookupEnv(testutil.EnvKeyUseBlockAdapter)
 	}
+	if blockstoreType == "" {
+		blockstoreType = "mem"
+	}
 	blockAdapter = testutil.NewBlockAdapterByType(t, &block.NoOpTranslator{}, blockstoreType)
-	cataloger, err := catalogfactory.BuildCataloger(conn, nil)
+	cfg := config.NewConfig()
+	cfg.Override(func(configurator config.Configurator) {
+		configurator.SetDefault(config.BlockstoreTypeKey, "mem")
```

Comment:
```suggestion
 		configurator.SetDefault(config.BlockstoreTypeKey, blockstoreType)
```
?

But if so, why does `blockstoreType` come from the environment?  Anyway current code seems inconsistent.